index.html

```diff
@@ -53,6 +54,246 @@ <h1>I'm Redwan.<br/>
+                    website. Look through some of my work 
+                    and experience! If you like what you see 
+                    and have a project you need coded, 
+                    don’t hesitate to contact me.
+                </p>
+            </div>
+            <!-- End of header top -->
+            <div class="headerBottom-bot">
+                <a href="#" class="link-text">LET’S CONNECT</a>
+                <div class="link-btn">
+                    <ul>
+                        <li>
+                            <a href="#"><img src="images/link-Vector4.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Instagram2.png" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/link-Vector5.svg" alt=""/></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="images/Twitter.svg" alt="Twitter"/></a>
+                        </li>
+                        
+                        <li>
+                            <a href="#"><img src="images/link-Vector3.svg" alt=""/></a>
+                        </li>
+                        
+                       
+                    </ul>
+                </div>
+                <div class="action">
+                    <button class="submit">Get my resume</button>
+                </div>
+                <div class="tags-bottom">
+                    <div class="container-links">
+                        <ul>
+                            <li class="lang"> 
+                                <div class="blink-name">Languages</div>
+                                <div class="blink-arrow"> <img class="Arrows" src="images/Down.png" alt=""></div> 
+                            </li>
+                            <li class="links">
+                                    <div class="btn">
+                                        <div class="link-log"><img src="images/JS.png" alt="">  </div>
+                                        <div class="lnk-txt"><a  href="">JavaScript</a></div> 
+                                    </div>
+                            </li>
+                            <li class="links" >
+                                <div class="btn">
+                                    <div class="link-log"><img src="images/HTML.png" alt="">  </div>
+                                    <div class="lnk-txt"><a  href="">HTML</a></div> 
+                                </div>
+                            </li>
+                            <li class="links" >
+                                <div class="btn">
+                                    <div class="link-log"><img src="images/CSS.png" alt="">  </div>
+                                    <div class="lnk-txt"><a  href="">CSS</a></div> 
+                                </div>
+                            </li>
```

Comment:
- [ ] Kindly note that html, CSS and javascript is a list under the languages. Kindly wrap them in a separate unordered list under the language list item. Your list of technologies sections should in its basic form look like this.

```html
<ul>
  <li>Languages
    <ul>
      <li>html</li>
      <li>javascript</li>
      <li>css</li>
    </ul>
  </li>
  <li>Frmaeworks</li>
  <li>Skills</li>
</ul>
```

Reply:
The initial and second reviewers emphasized the importance of using appropriate semantic tags for these elements.

For the Languages element, the presented technologies (HTML, JavaScript, and CSS) should be nested inside it as a list, rather than being sibling elements.

- [ ] In order to adhere to the semantic tags and maintain the HTML structure, let's incorporate the code snippet provided by the previous reviewer into our code.